guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -4,20 +4,11 @@
 Perform the following procedure if you want to disable {Cockpit} on {Project}.
 
 .Procedure
-. Run this `{foreman-installer}` command:
+* Run this `{foreman-installer}` command:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --no-enable-foreman-plugin-remote-execution-cockpit
-----
-. In the {ProjectWebUI}, navigate to *Administer* > *Settings* and click the *Remote execution* tab.
-. In the *Cockpit URL* row, erase the setting under *Value* and click *Submit*.
-This removes the *Web Console* button from the {ProjectWebUI}.
-. Uninstall the {Cockpit} package from {Project}:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# {package-remove} rubygem-foreman_remote_execution-cockpit
+# {foreman-installer} --foreman-plugin-remote-execution-cockpit-ensure absent
 ----
 +
```

Comment:
```suggestion

```

With `+`, the IMPORTANT box is part of the step to run foreman-installer. I'm not sure if that's something users would notice but it affects the indentation on the page.